packages/lit-html/src/lit-html.ts

```diff
@@ -523,18 +536,22 @@ const getTemplateHtml = (
           (attrNameEndIndex === -2 ? (attrNames.push(undefined), i) : end);
   }
 
+  let htmlResult: string|TrustedHTML =
+      html + (strings[l] || '<?>') + (type === SVG_RESULT ? '</svg>' : '');
+  if (policy) {
```

Comment:
I would prefer `(policy !== undefined)` if it doesn't add too many bytes. We use `!== undefined` enough that it should compress very well.

Reply:
Done